experimental/sdk/webpubsub-socketio-extension/examples/chat-with-negotiate/public/main.js

```diff
@@ -35,12 +35,15 @@ async function main(username) {
   if (!negotiateResponse.ok) {
     console.log("Failed to negotiate, status code =", negotiateResponse.status);
     return ;
-  } else {
-    endpoint = (await negotiateResponse.json()).url;
-    console.log("endpoint=", endpoint);
   }
+  const json = await negotiateResponse.json();
+  endpoint = json.endpoint;
+  path = json.path;
+  token = json.token;
```

Comment:
Do you use `token`?

Reply:
I don't think curren sample does, just put it here FYI.